config/docker/mysql/docker-compose.yml

```diff
@@ -1,28 +1,11 @@
-version: '2'
-
-services:
```

Comment:
You can probably just delete the whole mysql service, or just leave it in but don't use it.